src/Featurizers/Components/UnitTests/CMakeLists.txt

```diff
@@ -42,6 +44,31 @@ foreach(_test_name IN ITEMS
     #   - TimeSeriesFrequencyEstimator
     #   - TimeSeriesMedianEstimator
 )
+
+# GrainEstimatorImpl_UnitTest causes a compile error under the scenario:
+#
+#   - Windows (1)
+#   - Clang 8 (2)
+#   - x86 (3)
+#   - Release (4)
+#
+# disable that test for now.
+
+if(
+    WIN32                                                                       # 1
+    AND "$ENV{DEVELOPMENT_ENVIRONMENT_CPP_COMPILER_NAME}" STREQUAL "Clang-8"    # 2
+    AND "$ENV{DEVELOPMENT_ENVIRONMENT_CPP_ARCHITECTURE}" STREQUAL "x86"         # 3
+    AND "${CMAKE_BUILD_TYPE}" MATCHES "^Release.*$"                             # 4
+)
+    # TODO: Test this with Clang 8.0.1

```

Comment:
Didn't you mention yesterday that the bug still happens with 8.0.1?

Reply:
I did more research, and it happens with clang 8.0.0 and 8.0.1, but not with clang 9.0.0. I will update the comments accordingly.